lib/plugin.js

```diff
@@ -226,28 +232,58 @@ internals.implementation = function (server, options) {
 };
 
 
+const getTokenFromCookie = (cookieHeader, cookieName) => {
+
+    if (!(cookieHeader || cookieName)) {
+        return null;
+    }
+
+    const cookies = cookieHeader.split(/;(\s+)?/);
+
+    // find the cookie with the cookie name given in the settings
+    for (let i = 0; i < cookies.length; i += 1) {
+        const [name, value] = cookies[i].split('=');
+
+        if (name === cookieName) {
+            return value;
+        }
+    }
+
+    return '';
```

Comment:
`null` seems a more appropriate value to return to to signal it was not found.

Reply:
Yes, my bad. done